.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+
+name: is_dark
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: ['ubuntu-20.04']
+        ruby:
+          - 3.0
+          - 3.1
+          - 3.2
+          - 3.3
+          - head
+
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+
+    steps:
+      - name: Change ImageMagick policy to allow pdf->png conversion.
+        run: sudo sed -i 's/^.*policy.*coder.*none.*PDF.*//' /etc/ImageMagick-6/policy.xml
```

Comment:
Otherwise, we get an error in CI.
I looked at the solution here
https://wessman.co/github-actions-imagemagick

Reply:
Successfully running of test in CI https://github.com/Lyams/is_dark_ruby_gem/actions/runs/12539302987